doc/release/RELEASE-NOTES.md

```diff
@@ -23,9 +23,9 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-1069](https://opensupplyhub.atlassian.net/browse/OSDEV-1069) - The following changes have been made:
     * Changed the Postgres Docker image for the database to use the official one and make the local database setup platform-agnostic, so it doesn't depend on the processor architecture.
     * Built the PostGIS program from source and installed it to avoid LLVM-related errors inside the database Docker container during local development.
+* Make tile generation endpoint transaction-less and remove `CREATE TEMP TABLE` statement.
+* Added command `reindex_database`.
 * [OSDEV-1089](https://opensupplyhub.atlassian.net/browse/OSDEV-1089) Change format gunicurn logs not pass IP address to AWS CloudWatch.
-* Added command `reindex_database`
```

Comment:
Should we remove it from Release-Notes?

Reply:
Why?

Reply:
It came from another PR and I fixed conflict. So we need it I think